### DISCUSSION TITLE
Need clarification on 1.4.11 SC for graphical object

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3070] [Q&A] [answered]
I have a question about the chart.
Do the lines of the graph need to meet 3:1 contrast with the background?

For example, the line that represents the U.S. government information lacks a 3:1 contrast with the background.

![image](https://user-images.githubusercontent.com/115781693/221415379-d93f69c1-7af3-46c4-b83f-a659796c6f75.png)

**Comment by JAWS-test** [comment]
If there is no alternative in text form (e.g. a table with all the info from the chart in numerical form): yes, absolutely. Furthermore, all colors of the lines - each among each other - must keep a contrast of 3:1 or the lines must be distinguishable (e.g. direct labeling or different line shapes).

If there is an alternative in text form: Then here, if I remember old discussions correctly, say that the table is sufficient and others say that this is not the case because relevant information can be captured better graphically than in table form

**Reply by JAWS-test** [reply]
See https://github.com/w3c/wcag/issues/2690

https://www.w3.org/WAI/WCAG21/Techniques/general/G111 and https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

**Comment by patrickhlauke** [comment, marked as answer]
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#graphical-objects

> In order to understand the graph you need to discern the lines and shapes for each condition.

**Comment by Myndex** [comment]
While the one cyan line is a bit low in contrast for AA, the bigger issue, as you can see from the example below, is that this is a significant fail for A, 1.4.1 which requires that color not be the only way that information is conveyed.

In the following image the upper left sample is standard vision, the two samples on the right are common CVD types. Looking at the deuteranopia and protanopia, notice that you cannot distinguish most of the lines from each other, or identify them clearly per the legend.

While there is luminance variation, luminance is a poor way to encode information, that is for data coding like corresponding lines on a chart, two and at the most three variations if relying on luminance.

The lines need something more: occasional markers or dotted/dashed/etc.

<img width="1047" alt="Screen Shot of chart processed for CVD showing a lack of differentiation of the lines. Simulation from myndex.com/CVD/" src="https://user-images.githubusercontent.com/42009457/227727390-4c5a2f5d-97b0-4831-a44c-e796bcd2d2d2.png">

Simulation from [myndex.com/CVD/](https://myndex.com/CVD/)
